module3/Projects/PairProject.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: page
+title: Mod3 Pair Project
+---
+
+# Coffee Shop
+
+## Kick Off
+
+[Learning to Pair](https://docs.google.com/presentation/d/1iSdA31sgik4ejeB95R4t2kvmh1p0rANC7VAfjc2Wgmo/edit#slide=id.gc6f90357f_0_0)
```

Comment:
This slide deck is an excellent find!!

Reply:
I think we can keep this as is. I only have a couple of changes I think might help for our students. 

I think we should add time on the calendar for the pairs to check in and fill out that section.  Maybe half way through the work time? And then remove this question "When will we check in again about our process and progress?".

Also, I think we should add a specific place for each partner to put their answers in the "Before you start" section. Because I think we do want each partner to include their answers in the doc.

Finally, I think we should make sure to have them send a copy of their DTR document to instructors.

Reply:
yes, yes, and yes 😄